plugin/documents.py

```diff
@@ -522,7 +522,6 @@ def _on_query_completions_async(self, promise: sublime.CompletionList, location:
         if not session:
             resolve(promise, [])
             return
-        self._apply_view_settings(session)
```

Comment:
Nice one. Was bothered sometimes that the first autocomplete request is a bit wonky.